src/scss/vendors/_vendors.scss

```diff
@@ -1 +1,46 @@
 // Vendor-specific styles.
+
+//UIO
+
+//toc
```

Comment:
I believe it's decided that styling for table of content will be addressed separately when the design is in place. This toc block can be removed from this pull request.